renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
```

Comment:
For those curious: I believe `config:recommended` is documented here: https://docs.renovatebot.com/presets-config/#configrecommended